app/commands/user/reputation_token/award_for_pull_request.rb

```diff
@@ -34,19 +35,17 @@ def award_reputation_to_author
       def award_reputation_to_reviewers
         return unless just_closed?
 
-        reviews = octokit_client.pull_request_reviews(repo, pr_id)
-        reviewer_usernames = reviews.map { |reviewer| reviewer[:user][:login] }
+        reviewer_usernames = reviews.map { |reviewer| reviewer[:user][:login] }.uniq
+        reviewer_usernames.delete(github_username) # Don't award reviewer reputation to the PR author
```

Comment:
I've removed the PR author here as it could other be listed as a missing reviewer, but we're not processing that user as a reviewer anyway so that was slightly odd.